metadata-ingestion/src/datahub/ingestion/sink/file.py

```diff
@@ -27,6 +27,14 @@ def _to_obj_for_file(
 ) -> dict:
     if isinstance(obj, MetadataChangeProposalWrapper):
         return obj.to_obj(simplified_structure=simplified_structure)
+    elif isinstance(obj, MetadataChangeProposal) and simplified_structure:
+        serialized = obj.to_obj()
+        if serialized.get("aspect") and serialized["aspect"].get("contentType") in [
+            "application/json",
+            "application/json-patch+json",
```

Comment:
can we extract these into constants?

we already have `_ASPECT_CONTENT_TYPE` somewhere

Reply:
Yeah I was using that previously but I didn't really get what it meant. I can abstract them into some basic constants like `JSON_CONTENT_TYPE` and `JSON_PATCH_CONTENT_TYPE` though